docs/evaluate/temporal-cloud/overview.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: overview
+title: Overview - Temporal Cloud
+sidebar_label: Overview
+description: Temporal Cloud is a SaaS platform managing the durability of your Temporal Applications, implemented using Temporal SDK. It supervises workflow histories and supports multiple isolated namespaces.
+slug: /cloud/overview
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - term
+tags:
+  - explanation
+  - term
+---
+
+[Temporal Cloud](https://temporal.io/cloud) serves as a software-as-a-service (SaaS) infrastructure platform, specifically designed to manage the durability of your Temporal Applications.
```

Comment:
Do we hyphenate this?  There's a lot of different ways, and I'll admit I don't know ours. If we do, just decline this suggestion. 

```suggestion
[Temporal Cloud](https://temporal.io/cloud) serves as a software as a service (SaaS) infrastructure platform, specifically designed to manage the durability of your Temporal Applications.
```